app/containers/MassEnergizeSuperAdmin/Events/notifcation-settings/EventNotificationSettings.js

```diff
@@ -102,7 +102,17 @@ export default function EventNotificationSettings(props) {
     dispatch(loadAllEvents(updatedEvents));
   };
 
+  const readyForFlight = () => {
+    if (!targetCommunities?.length) {
+      notify("Please select the communities that these settings apply to");
+      return false;
+    }
+
+    return true
+  };
+
   const sendChangesToBackend = () => {
+    if (!readyForFlight()) return;
     setLoading(true);
     setNotification({});
     const isALL = targetCommunities?.find((c) => typeof c === "string" && c?.toLowerCase() === "all");
```

Comment:
can you use a more descriptive names here to make the code more self documenting?
The function name `readyForFlight` doesn't seem to tell what it does, neither does the variable name `c` on line 118